gui/confirm.lua

```diff
@@ -1,7 +1,7 @@
 -- confirm plugin options
 --[====[
 
-gui/confirm-opts
+gui/confirm
 ================
 A basic configuration interface for the `confirm` plugin.
 
```

Comment:
you can remove this old copy of the docs. it's no longer used and is just liable to get out of sync with the real docs in the rst